### DISCUSSION TITLE
Unable to deploy to heroku - Quasar not found

### DISCUSSION THREAD

**Discussion by realtebo** [quasarframework/quasar#14449] [General - Components / Directives / etc]
When pushing to heroku master I see all working until 

```
remote:        $ yarn && yarn build
remote:        [1/5] Validating package.json...
remote:        [2/5] Resolving packages...
remote:        [3/5] Fetching packages...
remote:        [4/5] Linking dependencies...
remote: warning " > @intlify/vite-plugin-vue-i18n@3.4.0" has unmet peer dependency "vite@^2.0.0".
remote:        [5/5] Building fresh packages...
remote:        $ quasar build
remote: /bin/sh: 1: quasar: not found
remote: error Command failed with exit code 127.
remote:        info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
remote: error Command failed with exit code 127.
remote:        info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
remote:
remote: -----> Build failed
remote:
```

I have no idea on how to fix. 

I am following exactly instructions at: 

https://quasar.dev/quasar-cli-vite/developing-spa/deploying#deploying-with-heroku

**Comment by MilosPaunovic** [comment]
From your code it seems like you're running this on some kind of CI/CD and it seems like you don't have `@quasar/cli` installed there. So, you need to install it first before running any `quasar ...` command.

**Reply by realtebo** [reply]
> you're running this on some kind of CI/CD 

No, I'm simply  following instructions on the page I linked. No CI/CD. only heroku cli on my windows system.

**Comment by pdanpdan** [comment]
Or you can replace the use of `quasar ...` with `yarn quasar ...` or the npm equivalent one (so you use the local quasar)

**Reply by realtebo** [reply]
both not working

**Comment by realtebo** [comment]
Fixed adding a 

    yarn global add quasar

before every other command in the postbuild script

I think someone should try and fix the doc if my problem is reproducible

**Comment by yusufkandemir** [comment]
Please share your whole `package.json` file.
